### PR TITLE
MLR-274: change name of env variable to match oauth naming convention

### PR DESCRIPTION
**Pull request by kmschoep-usgs**
wondering if this entire config.py file should be moved to the docker repo?

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def get_roles(dict):
 JWT_ROLE_CLAIM = get_roles
 
 # Set the path
-AUTH_CERT_PATH = os.getenv('auth_cert_path', True)
+AUTH_CERT_PATH = os.getenv('oauthClientSecret', True)
```
Comment (zmoore-usgs) [inline config.py]: Is this where the client secret is going? The naming seems to lean more towards this being the SSL cert used by Water Auth but perhaps it's just confusing naming

**Comment by zmoore-usgs** [comment]
I don't think I know enough about how the python services work to know if moving the config.py file is a good idea or not.

File: config.py (hunk 1 above)
Comment (kmschoep-usgs) [reply config.py]: I'm sure I'm the one who's confused.  It might be for the water auth cert?

**Comment by kmschoep-usgs** [comment]
I"ll leave config.py in.  It's pulling the env variables to use at least some of them in the app.

**Pull request merged**